source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_visitors.jsonnet

```diff
@@ -22,6 +22,37 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
         },
       ],
     },
+    definitions: [
```

Comment:
The spreadsheet says to also remove the `guidance`.

Reply:
Same as above, got pasted back during merge. Thanks👍